examples/todomvc/components/App.vue

```diff
@@ -13,10 +13,11 @@
     </header>
     <!-- main section -->
     <section class="main" v-show="todos.length">
-      <input class="toggle-all"
+      <input class="toggle-all" id="toggle-all"
```

Comment:
Yes, it's not. The checkbox arrow(`.toggle-all`) need a label tag next to it, and can't be wrapped with label, because it was written like this in todomvc-app-css:
```css
.toggle-all + label:before {
    content: '❯';
    font-size: 22px;
    color: #e6e6e6;
    padding: 10px 27px 10px 27px;
}
```
Then if i want to click the arrow and trigger the event, need to bind the `label` tag to input with the `for` attribute, so i set id on input. Is there any other way better than this?